src/AzureRMAssets.ts

```diff
@@ -73,83 +73,41 @@ export class AzureRMAssets {
      */
     public static getFunctionMetadataUri(): Promise<string> {
         if (AzureRMAssets._functionMetadataUri === undefined) {
-            AzureRMAssets._functionMetadataUri = AzureRMAssets.getAssetUri("ExpressionMetadata.json");
+            AzureRMAssets._functionMetadataUri = AzureRMAssets.getLocalAssetUri("ExpressionMetadata.json"); //AzureRMAssets.getAssetUri("ExpressionMetadata.json");
         }
         return AzureRMAssets._functionMetadataUri;
     }
 
-    private static getAssetUri(assetFileName: string): Promise<string> {
-        return AzureRMAssets.getCurrentVersionRedirectUri()
-            .then((versionRedirectUri: string) => {
-                return versionRedirectUri + (versionRedirectUri.endsWith("/") ? "" : "/") + assetFileName;
-            });
+    private static getLocalAssetUri(assetFileName: string): Promise<string> {
+        return Promise.resolve<string>(path.join(__filename, "..", "..", "..", "assets", assetFileName));
+    }
+
+    private static readFile(path: string): Promise<string> {
```

Comment:
Not necessary for this PR, but I'd take a look at fs-extra. It's got promisified versions of all the fs functions (plus some other useful stuff)